CONTRIBUTING.md

```diff
@@ -35,17 +36,17 @@ If you encounter any bugs or issues with the project, please [create a new issue
 
 ### Feature Requests
 
-If you have a feature idea that you would like to see implemented, please [create a new issue](../../issues/new?assignees=&labels=enhancement%2Cneeds-triage&projects=&template=feature.yaml&title=(feature)+<title>) with the following information:
+If you have a feature idea that you would like to see implemented, please [create a new issue](<../../issues/new?assignees=&labels=enhancement%2Cneeds-triage&projects=&template=feature.yaml&title=(feature)+<title>>) with the following information:
```

Comment:
**🔨 issue:** ‏Why have you changed this file? Your changes are not necessary here.